src/app/modules/directives/numbers-only.directive.ts

```diff
@@ -11,11 +11,18 @@ import { El } from './directives.interface';
 })
 export class NumbersOnlyDirective {
```

Comment:
selector: '[intrNumbersOnly]' is your directive may work with any another selector?